pyproject.toml

```diff
@@ -342,6 +342,7 @@ addopts = ["--numprocesses=logical"]
 
 [tool.mypy]
 warn_unused_ignores = true
+pretty = true
```

Comment:
Only recently discovered this https://mypy.readthedocs.io/en/stable/config_file.html#confval-pretty

It underlines the code that caused the error, which is pretty handy IMO

Reply:
That's nice! :)